lib/browser/guest-view-manager.js

```diff
@@ -74,7 +74,7 @@ const createGuest = function (embedder, params) {
 
   // Clear the guest from map when it is destroyed.
   guest.once('destroyed', () => {
-    if (Object.keys(guestInstances).includes(guestInstanceId)) {
+    if (guestInstances.hasOwnProperty(guestInstanceId)) {
```

Comment:
I have sad news, this breaks if `guestInstances` has a property named `"hasOwnProperty"`. It is, I think, _never_ correct to call `x.hasOwnProperty(y)`, always always use `Object.prototype.hasOwnProperty.call(x, y)`.
```suggestion
    if (Object.prototype.hasOwnProperty.call(guestInstances, guestInstanceId)) {
```

Reply:
Ah good catch 👍